frontend/client/src/screens/CodeValidations.js

```diff
@@ -24,7 +24,11 @@ const CodeValidationsBase = observer((props) => {
 
   const genNewCode = async () => {
     try {
-      let code = await props.store.getVerificationCode()
+      // TODO: testType and testDate should be set by fields in the interface; blocked awaiting v4 figma
+      let code = await props.store.getVerificationCode({
+        testType: 'confirmed',
+        testDate: new Date().toJSON().substring(0, 10),
```

Comment:
This looks ok to me, understanding that `testType` and `testDate` are both essentially hardcoded values to change later.